### ISSUE TITLE
Confusing/wrong error message when generating the baseline if the ruleset-file is missing

### ISSUE THREAD

**Issue by jakobbuis** [phpmd/phpmd#958]
- PHPMD version: 2.12.0
- PHP Version: 8.1.6
- Installation type: composer
- Operating System / Distribution & Version: Ubuntu 21.10

### Current Behavior
An error message in the console:
```
Unable to find the baseline file. Use --baseline-file to specify the filepath
```

### Expected Behavior
An error message stating the rule-set file is missing. 

### Steps To Reproduce:
This error got triggered in a roundabout way because I switched away from the git branch I was working on, thus removing the committed ruleset file from disk. When trying to establish the baseline, the error message is incorrect and leads users down the wrong path. It can be intentionally reproduced like thus:

1. Install PHPMD using composer
2. Create a valid ruleset file `phpmd.xml`. 
3. Run `vendor/bin/phpmd . ansi phpmd.xml --generate-baseline`. This should work, assuming you have at least one violation.
4. Remove the ruleset file `phpmd.xml` and the generated baseline file.
5. Run `vendor/bin/phpmd . ansi phpmd.xml --generate-baseline`. This fails with the mentioned error message that does not reflect the problem.

**Important caveat:** actually giving the command a baseline file location by running:
```
vendor/bin/phpmd . ansi phpmd.xml --generate-baseline --baseline-file=foo.xml
```
does not resolve the problem, making this one particularly confusing to users. 

## Checks before submitting
* [x] Be sure that there isn't already an issue about this. See: [Issues list](https://github.com/phpmd/phpmd/issues)
* [x] Be sure that there isn't already a pull request about this. See: [Pull requests](https://github.com/phpmd/phpmd/pulls)
* [x] I have added every step to reproduce the bug.
* [x] If possible I added relevant code examples.
* [x] This issue is about 1 bug and nothing more.
* [x] The issue has a descriptive title. For example:  "JSON rendering failed on Windows for filenames with space".
